v2/terraform_flannel.py

```diff
@@ -328,3 +328,13 @@ def down(self):
             self.deployer.down()
         except Exception as e:
             raise e
+
+    def collectWindowsLogs(self):
+        self.opts.workspace = "/tmp/k8s-ovn-ovs"
+        collect_logs_script = os.path.join(self.opts.workspace, "v2", "collect-logs.ps1")
```

Comment:
If we need self.opts.workspace to determine where to run collect-logs from, we can either use cwd or we can use pass self.opts.workspace as a CLI arg properly, from bootstrap.